include/ISymbolManager.h

```diff
@@ -42,6 +42,7 @@ class ISymbolManager {
 	virtual void loadSymbolFile(const QString &filename, edb::address_t base)          = 0;
 	virtual void setSymbolGenerator(ISymbolGenerator *generator)                       = 0;
 	virtual void setLabel(edb::address_t address, const QString &label)                = 0;
+    virtual void restoreLabels(const QVariantList &labels)                             = 0;
```

Comment:
Here and in many other places: please note that EDB uses 4-char tabs, not spaces, for indentation.

Reply:
OK! Sorry for that, I opened the EDB source code today, for the first time. I started this patch using nano with a specific profile for Python and switched to QtCreator.

I will correct that during the next commits.

Thank you for your quick review!

Reply:
Resolved with the last commit.